### ISSUE TITLE
Brand Footer: Do not include the title attribute unless having additional information appear upon hover is essential.

### ISSUE THREAD

**Issue by McManning** [osuresearch/ui#90]
The OSU logo image in the footer contains both alt text and a title. Screen readers announce both, leading to redundancy in the announcement.  
•	This is a WCAG Best Practice
•	Recommendation: Remove titles from images unless it provides essential additional information that cannot be delivered any other way.

**Comment by McManning** [comment]
For context:
![image](https://github.com/osuresearch/ui/assets/1057607/759d3d77-cbe7-488f-9913-26c75be338d4)
